transport_plugins/bled112/blob/requirements.txt

```diff
@@ -0,0 +1,3 @@
+iotile-core>=5.0.0,<6
+pyserial>=3.4.0,<4
+iotile_transport_bled112
```

Comment:
Since you specified versions for the other two, we should have it present for the transport as well

Reply:
Though maybe this doesn't make sense, since the script is part of the transport plugin?